app/client/components/user/SingleUserManagement.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from "react";
+import { connect } from "react-redux";
+
+import { editUser, resetPin } from "../../reducers/userReducer";
+import { TransferUsage } from "../../reducers/transferUsage/types";
+import { ReduxState } from "../../reducers/rootReducer";
+import EditUserForm, { ICreateUserUpdate } from "./EditUserForm";
+
+interface DispatchProps {
+  editUser: (body: any, path: any) => void;
```

Comment:
isn't path `string`?

Reply:
also might not hurt to have something like ```
interface IJSONSerialisedUserData {
...
}
```

though i can see how that could be a lot of work

Reply:
mhm I think it's OK as it is

Reply:
Once the UserReducer is refactored into Typescript then we can import the `EditUserPayload` from `auth/types.ts`

Reply:
oh yeah that's true!